docs/source/_toctree.yml

```diff
@@ -254,8 +254,6 @@
       title: MegatronBERT
     - local: model_doc/megatron_gpt2
       title: MegatronGPT2
-    - local: model_doc/mluke
-      title: MLUKE
```

Comment:
This one should stay and the next one should be removed I beleive, if we keep alphabetical order.